WireUI/Sources/WireIndividualToTeamMigrationUI/Views/Previews/ConfirmationPreview.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireFoundation
+
+#Preview {
```

Comment:
I'm wondering why you put all the previews in the `Previews` folder? Is this a best practice? Doesn't it make it harder to work on the view?

Reply:
What I did a couple of times so far is putting the setup code for the previews in a different file and kept the `#Preview` macro in the file of the view to preview.

Reply:
I looked at what was already in this module, and noticed what had been done in WireSidebarUI. I personally have no preference whatsoever.

Reply:
👍I would go with @caldrians suggestion. I will do the same the next time I work on a swiftUI view.

Reply:
Done